js/esbuild.js

```diff
@@ -0,0 +1,72 @@
+const argv = require('yargs').argv;
+const { watch } = argv;
```

Comment:
yargs is a little heavy for what we need here- could use `const watch = process.argv.includes('--watch');`

Reply:
updated